docs/en/modules/rtctime.md

```diff
@@ -67,6 +67,34 @@ For applications where it is necessary to take samples with high regularity, thi
 rtctime.dsleep_aligned(5*1000000, 3*1000000)
 ```
 
+## rtctime.epoch2cal()
+
+Converts a Unix timestamp to calendar format.
```

Comment:
It's pretty obvious that you'll get a UTC calendar but I'd like to see this mentioned here nonetheless just to avoid potential questions on one of our support channels.